app/(discover)/discover/utils.ts

```diff
@@ -2,7 +2,7 @@ import { TagMap } from "@/types"
 
 import { createProxyUrls } from "@/app/utils"
 
-export const fetchMix = async (tags: TagMap, limit: number) => {
+export const fetchMix = async (tags: string[], limit: number) => {
```

Comment:
Ideally the type of tags would be consistent everywhere

Reply:
Agreed, Looks like the places TagMap is used is the [TagList ](https://github.com/State-Channel-4/frontend/blob/feat/browse-topic/components/ui/tag-list.tsx) component and [TagRow](https://github.com/State-Channel-4/frontend/blob/feat/browse-topic/components/ui/tag-row.tsx). Was under the impression these would no longer be needed in the new design so maybe we just remove them?